### DISCUSSION TITLE
can i load XML-file from classic shell?

### DISCUSSION THREAD

**Discussion by bmw216** [Open-Shell/Open-Shell-Menu#1874] [Q&A] [answered]
can i load XML-file from classic shell 4.3.1 on windows 10 into open shell windows 11?

**Comment by ge0rdi** [comment, marked as answer]
Yup, there should be no problem with that.
`Open-Shell` uses the same settings (plus some newly added).

**Reply by bmw216** [reply]
does open shell also get back the classic right-click menu on files and desktop?

**Reply by ge0rdi** [reply]
Nope.

**Reply by bmw216** [reply]
Open shell should fix that too. Very important.

How we get back right click menu?

**Reply by ge0rdi** [reply]
> How we get back right click menu?

https://www.howtogeek.com/759449/how-to-get-full-context-menus-in-windows-11s-file-explorer/
